packages/tsconfig-reference/copy/id/options/disableSourceOfProjectReferenceRedirect.md

```diff
@@ -0,0 +1,7 @@
+---
+display: "Nonaktifkan Pengalihan Referensi Proyek Sumber"
+oneline: "Gunakan berkas d.ts sebagai sumber kebenaran untuk perkakas antara batasan proyek gabungan"
+---
+
+Saat bekerja dengan [proyek TypeScript komposit](/docs/handbook/project-references.html), opsi ini menyediakan cara untuk kembali [kembali ke pra-3.7](/docs/handbook/release-notes/typescript-3-7.html#build-free-editing-with-project-references) perilaku di mana berkas d.ts digunakan sebagai batas antar modul.
+Di 3.7 sumber kebenaran sekarang adalah berkas TypeScript Anda.
```

Comment:
Kalau `sumber kebenaran` itu konteks dalam bahasa Inggrisnya apakah?

Reply:
`Source of truth`

Reply:
Cocoknya gmn mas?
Mungkin ada masukan dari temen" yg lain?

Reply:
Hmm saya juga belum nemu padanan yg cocok sih, mungkin setelah `Di 3.7` itu diberi koma biar lebih enak dibaca, gimana Mas?

Reply:
Iya klo ditambahin koma sih enak bacanya mas

Jadi `Di 3.7, sumber kebenaran adalah berkas Typescrip Anda`

Reply:
Sumber kebenaran oke kok, saya sudah sering pakai istilah itu juga